litestar/security/jwt/auth.py

```diff
@@ -692,35 +611,6 @@ class OAuth2PasswordBearerAuth(Generic[UserType, TokenT], BaseJWTAuth[UserType,
     ``accepted_audiences`` is a sequence of length 1
     """
 
-    @property
-    def middleware(self) -> DefineMiddleware:
```

Comment:
backwards compatibility; see earlier comment

Reply:
done on the base too